### ISSUE TITLE
Remove Extra Files from Repo Tracking

### ISSUE THREAD

**Issue by ArikVhedrovix** [NSUCIS/CommunicationProject#2]
Currently a lot of extra files get marked as changed ever time the project is run. This will clutter the commits and make it hard to see what was actually changed by the programmer. To fix we need to use the command on each file that needs to be removed:
`git rm --cached <filename>`

**Comment by ArikVhedrovix** [comment]
@ChristopherNYoung I can do this if you can help me figure out which files (beside .vb) are actually necessary to keep in the source code.

**Comment by ChristopherNYoung** [comment]
Yeah I'm off today after this class, I'll look at it when I get home
